bytecomp/translmod.ml

```diff
@@ -58,10 +58,12 @@ let transl_extension_constructor env path ext =
   in
   match ext.ext_kind with
     Text_decl(args, ret) ->
-      Lprim(prim_set_oo_id,
-            [Lprim(Pmakeblock(Obj.object_tag, Mutable),
-                   [Lconst(Const_base(Const_string (name,None)));
-                    Lconst(Const_base(Const_int 0))])])
+      let id = Ident.create "set_oo_id" in
+      Llet(Strict, id,
+           Lprim(Pmakeblock(Obj.object_tag, Mutable),
+                 [Lconst(Const_base(Const_string (name,None)));
+                  Lconst(Const_base(Const_int 0))]),
+           Lsequence(Lprim(prim_set_oo_id, [Lvar id]), Lvar id))
   | Text_rebind(path, lid) ->
```

Comment:
From what I can tell, this file change[bytecomp/translmod.ml) is unnecessary ( correct me if  I am wrong).
Is it possible to leave it untouched? The old version is straightforward to be optimized and inlined in my JS backend, here an `Strict` binding is introduced which makes it a bit tricky to do a general optimization, of course, I can do an ad-hoc peephole, but it would be nice to avoid too many ad-hoc optimizations, thanks